letsencrypt/README.rst

```diff
@@ -97,6 +114,13 @@ an upstream for your odoo instance and do something like::
         proxy_pass    http://yourodooupstream;
     }
 
+If you're using a multi-database installation (with or without dbfilter option)
```

Comment:
I don't see that the certificates/keys are going to be overwritten. Where do you see that?